src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -4441,6 +4449,12 @@ namespace FanCoilUnits {
         FanCoil(FanCoilNum).SensCoolEnergy = FanCoil(FanCoilNum).SensCoolPower * ReportingConstant;
         FanCoil(FanCoilNum).TotCoolEnergy = FanCoil(FanCoilNum).TotCoolPower * ReportingConstant;
         FanCoil(FanCoilNum).ElecEnergy = FanCoil(FanCoilNum).ElecPower * ReportingConstant;
+
+        if (FanCoil(FanCoilNum).FirstPass) { // reset sizing flags so other zone equipment can size normally
+            if (!DataGlobals::SysSizingCalc) {
+                DataSizing::resetHVACSizingGlobals(DataSizing::CurZoneEqNum, 0, FanCoil(FanCoilNum).FirstPass);
```

Comment:
Because some parent models do not call the coils from the sizing routine. Then after sizing (and Init) is completed the coils are called from the Sim routine. So the reset needs to happen after the coils are called.